app/src/main/java/com/example/homework1calculator/Utils.kt

```diff
@@ -1,5 +1,14 @@
 package com.example.homework1calculator
 
 
-
-
+val Char.isCorrectSymbol
+    get() = when (this) {
+        '*' -> true
+        '/' -> true
+        '+' -> true
+        '-' -> true
+        '.' -> true
+        '(' -> true
+        ')' -> true
+        else -> false
+    }
```

Comment:
копипаст, можно несколько веток через запятую указывать, либо использовать if 
```suggestion
val Char.isCorrectSymbol
    get() =  this in availableChars
```